### ISSUE TITLE
Can't create new contacts on master

### ISSUE THREAD

**Issue by schiessle** [nextcloud/contacts#29]
Since the update to sabreDAV 3.2 on master (https://github.com/nextcloud/server/pull/1792)  I can't create new contacts.

cc @rullzer maybe you have a idea how to fix is, I saw that you also fixed other issues after the sabreDAV update

**Comment by nickvergessen** [comment]
```xml
<?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:exception>Sabre\DAV\Exception\UnsupportedMediaType</s:exception>
  <s:message>Validation error in vCard: The FN property must appear in the VCARD component exactly 1 time</s:message>
</d:error>
```

**Comment by nickvergessen** [comment]
VCard sent is:
```
BEGIN:VCARD
VERSION:3.0
UID:bac8b1b9-a123-4ca2-b1ae-593d6c995814
END:VCARD
```

**Comment by LukasReschke** [comment]
Isn't this https://github.com/nextcloud/contacts/pull/8 ?

**Comment by LukasReschke** [comment]
The PR seems to miss some commits though. See https://github.com/owncloud/contacts/pull/526/files directly

**Comment by rullzer** [comment]
Fixed by #8
